src/main/java/io/jenkins/plugins/opentelemetry/job/log/BuildInfo.java

```diff
@@ -5,38 +5,70 @@
 
 package io.jenkins.plugins.opentelemetry.job.log;
 
+import com.google.common.base.Objects;
 import io.jenkins.plugins.opentelemetry.semconv.JenkinsOtelSemanticAttributes;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.context.Context;
+import org.yaml.snakeyaml.nodes.CollectionNode;
```

Comment:
useful? I can't find references in the code

Reply:
IO have to clean imports